src/mail/view/MinimizedMailBar.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import m from "mithril"
+import {WindowFacade} from "../../misc/WindowFacade"
+import {px} from "../../gui/size"
+import {MINIMIZED_HEIGHT, MINIMIZED_WIDTH, MinimizedMailElement} from "./MinimizedMailElement"
+import {MinimizedMailModel} from "../model/MinimizedMailModel"
+
+export type MinimizedMailBarAttrs = {
+	windowFacade: WindowFacade,
+	minimizedMailModel: MinimizedMailModel
+}
+
+/**
+ *  Bar that gets rendered at the bottom of the screen for 5 most recent minimized mails to be displayed in
+ *  Renders iff client is not mobile
+ */
+export class MinimizedMailBar implements MComponent<MinimizedMailBarAttrs> {
+	_windowCloseUnsubscribe: () => mixed
+	model: MinimizedMailModel
+
+	constructor(vnode: Vnode<MinimizedMailBarAttrs>) {
+		this._windowCloseUnsubscribe = () => false
+		this.model = vnode.attrs.minimizedMailModel
+	}
+
+	view(vnode: Vnode<MinimizedMailBarAttrs>): Children {
+		return m(".flex-end.abs", {
+				oncreate: () => this._windowCloseUnsubscribe = vnode.attrs.windowFacade.addWindowCloseListener(() => true),
+				onremove: () => this._windowCloseUnsubscribe(),
+				style: {
+					bottom: 0,
+					height: px(MINIMIZED_HEIGHT),
+					width: px((MINIMIZED_WIDTH * 5) + 40), // we allow 5 popups with a margin-right of 8px
```

Comment:
Maybe we should make this 5 a constant then? And compute this 40 instead of just hardcoding it?